examples/flipper/lib.rs

```diff
@@ -71,34 +71,4 @@ mod tests {
 
         Ok(())
     }
-
```

Comment:
This requires to implement the SandboxConfig trait manually, to keep things simple I am removing it here, the concrete example will live in ink!

Reply:
For reference this will be pretty much used like this 
```rust
    use drink::impl_sandbox_config;
    use pallet_contracts_mock_network::{
        parachain::Runtime as ParachainRuntime, ParaA, ALICE, EXT_PARAA,
    };
    use xcm_simulator::TestExt;

    struct ParachainConfig;
    impl SandboxConfig for ParachainConfig {
        type Runtime = ParachainRuntime;

        fn execute_with<T>(execute: impl FnOnce() -> T) -> T {
            ParaA::execute_with(execute)
        }

        fn dry_run<T>(action: impl FnOnce() -> T) -> T {
            EXT_PARAA.with(|v| {
                let backend_backup = v.borrow_mut().as_backend();
                let result = action();

                let mut v = v.borrow_mut();
                v.commit_all().expect("Failed to commit changes");
                v.backend = backend_backup;
                result
            })
        }

        fn register_extension<E: ::core::any::Any + Extension>(ext: E) {
            EXT_PARAA.with(|v| v.borrow_mut().register_extension(ext));
        }

        fn initialize_block(
            height: frame_system::pallet_prelude::BlockNumberFor<Self::Runtime>,
            parent_hash: <Self::Runtime as frame_system::Config>::Hash,
        ) {
            minimal::BlockBuilder::<ParachainRuntime>::initialize_block(height, parent_hash)
        }

        fn finalize_block(
            height: frame_system::pallet_prelude::BlockNumberFor<ParachainRuntime>,
        ) -> Result<<Self::Runtime as frame_system::Config>::Hash, String> {
            minimal::BlockBuilder::<ParachainRuntime>::finalize_block(height)
        }

        fn default_actor() -> AccountIdFor<Self::Runtime> {
            ALICE
        }

        fn get_metadata() -> RuntimeMetadataPrefixed {
            ParachainRuntime::metadata()
        }

        fn convert_account_to_origin(
            account: AccountIdFor<Self::Runtime>,
        ) -> <<Self::Runtime as frame_system::Config>::RuntimeCall as Dispatchable>::RuntimeOrigin
        {
            Some(account).into()
        }
    }
```